src/shared/header/hamburgerMenu/hamburgerMenuDirective.js

```diff
@@ -7,5 +7,23 @@ export default angular.module('shared')
             templateUrl: 'hamburgerMenu.html',
             controller: 'HamburgerMenuController',
             controllerAs: 'hamburger',
+            link: ($scope, iElem) => {
+                function closeMenu() {
+                    if (!$scope.hamburger.active) {
```

Comment:
please, rename active  to isActive to underline bool value(here and controller)

Reply:
Done